Packs/FeedMitreAttackv2/Integrations/FeedMitreAttackv2/FeedMitreAttackv2.py

```diff
@@ -301,12 +311,36 @@ def map_fields_by_type(indicator_type: str, indicator_json: dict):
             'stixaliases': indicator_json.get('x_mitre_aliases'),
             'stixdescription': indicator_json.get('description'),
             'operatingsystemrefs': indicator_json.get('x_mitre_platforms')
+        },
+        "Campaign": {
+            'description': indicator_json.get('description'),
+            'aliases': indicator_json.get('aliases')
         }
     }
     generic_mapping_fields.update(mapping_by_type.get(indicator_type, {}))  # type: ignore
     return generic_mapping_fields
 
 
+def extract_timestamp_from_description(description: str) -> str:
+    if description is None:
+        return ''
+    if 'Citation' in description:
+        return ''
+    match = re.search(r"\((.+)\)", description)
+    timestamp = match.group(1) if match else ''
+    return timestamp
+
+
+def get_TLP(indicator_json: dict) -> str:
```

Comment:
Please lower case the `TLP`.
```suggestion
def get_tlp(indicator_json: dict) -> str:
```